test/libsolidity/syntaxTests/largeTypes/large_mapping_in_array.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function f() public {
+        mapping (uint => uint[2][2**255])[][] storage A;
+        mapping (uint => uint[2][2**255])[2] storage B;
+    }
```

Comment:
I'd add some tests for oversized data in more situations:
- parameters and return values
- casts
- operator `new`
- `abi.decode()`